src/common/message.ts

```
@@ -20,3 +20,53 @@
+        };
+      } else {
+        return {
+          text: `${key}: ${value}`,
+        };
+      }
+    });
+  }
+}
+
+function createListChildren(object: unknown[] | object): string[] {
+  if (object instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)